content/components/button.mdx

```diff
@@ -428,6 +428,39 @@ Limit primary button usage. Only use one primary button on the page, whenever po
   </Dont>
 </DoDontContainer>
 
+### Selections
+
+Buttons are commonly used to show a choice within a `select`, [SelectPanel](/components/selectpanel) or [ActionMenu](/components/action-menu). In such instances, they are frequently paired with an internal or external label.
```

Comment:
> to show a choice within a `select`

When do we use a button to show the selections from a `select`?

Reply:
Actually I might be wrong on this one. I can remove that.